website/src/pages/use-cases/onboarding.tsx

```diff
@@ -0,0 +1,324 @@
+                                    to="/get-started"
+                                    title="Try Sourcegraph."
+                                >
+                                    Try Sourcegraph now
+                                </Link>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </>
+        }
+    >
+        <ContentSection className="my-lg-5">
+            <div className="row mx-lg-0 mx-4">
+                <div className="d-flex justify-content-center w-100 mt-7 mb-lg-4 mb-0">
+                    <h1 className="text-center font-weight-bold w-75 px-lg-8">
+                        Make your codebase accessible for your entire team
+                    </h1>
+                </div>
+                <div className="d-flex flex-column flex-lg-row mt-lg-4 mt-6 mb-6">
+                    <div className="text-center">
+                        <MagnifyIcon className="mb-4" color="#00a1c7" size={40} />
+                        <h4 className="font-weight-bold">Find answers across all repositories</h4>
+                        <p>
+                            Codebases grow increasingly complex over time. Sourcegraph enables developers to search
+                            everything at once without needing to clone and search locally.
+                        </p>
+                    </div>
+                    <div className="mx-lg-7 text-center">
+                        <UploadIcon className="mb-4" color="#00a1c7" size={40} />
+                        <h4 className="font-weight-bold">Share knowledge quickly with links to specific code</h4>
+                        <p>
+                            Knowledge sharing takes time. With Sourcegraph, developers can share links directly to
+                            specific lines of code and ask questions with context included.
+                        </p>
+                    </div>
+                    <div className="text-center">
+                        <CompassOutlineIcon className="mb-4" color="#00a1c7" size={40} />
+                        <h4 className="font-weight-bold">Navigate and understand large codebases</h4>
+                        <p>
+                            Make new codebases approachable, not aggravating. Search across all your repositories in one
+                            place with Sourcegraph's IDE-inspired features.
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </ContentSection>
+
+        <div className="bg-gradient-blue-mist">
+            <ContentSection className="my-7">
+                <div className="row flex-column flex-lg-row align-items-center justify-content-between">
+                    <div className="p-lg-0 col-lg-6 px-4">
+                        <h1 className="mb-4 font-weight-light">Developer onboarding is slow and expensive</h1>
```

Comment:
What do you think about making this `font-weight-bold` on this page and the vulns page for design consistency? We could also add `mw-400` here too.

Reply:
I am good with that!